scripts/dts/edtlib.py

```diff
@@ -1245,8 +1262,14 @@ def _init_regs(self):
                               .format(address_cells, size_cells)):
             reg = Register()
             reg.node = self
-            reg.addr = _translate(to_num(raw_reg[:4*address_cells]), node)
-            reg.size = to_num(raw_reg[4*address_cells:])
+            if address_cells == 0:
+                reg.addr = None
```

Comment:
The documentation for the `Register` class needs to be updated.

Reply:
done